### PR TITLE
Turns out that ast.Move is dead code and there's quite a lot of code that goes out with it

### PR DESCRIPTION
**Pull request by msackman-synthace**
Test Plan:
suitably set up antha-runner testbundles passes. That said, I'm not 100% sure if our test bundles would check tasks across different devices.

### CODE REVIEW

File: codegen/codegen.go [hunk 1]
```diff
@@ -499,30 +408,10 @@ func (a *ir) addMoves(ctx context.Context, t *target.Target) error {
...
-	order, err := graph.TopoSort(graph.TopoSortOpt{
+	_, err := graph.TopoSort(graph.TopoSortOpt{
 		Graph: a.DeviceDeps,
 	})
-	if err != nil {
-		return err
-	}
-
-	// Disable move pass because it is unused
-	if true {
```
Comment (msackman-synthace) [inline codegen/codegen.go]: This is why Move is dead code - we never actually call into addMove.

File: codegen/codegen.go [hunk 2]
```diff
@@ -366,130 +354,6 @@ func (a *ir) tryPlan(ctx context.Context) error {
...
-		insts = append(insts, ins...)
-	}
-
-	a.output[run] = append(insts, a.output[run]...)
-
-	return nil
-}
-
 // Add implied moves between devices
 func (a *ir) addMoves(ctx context.Context, t *target.Target) error {
```
[the remark this answers is not in the document]
Comment (msackman-synthace) [reply codegen/codegen.go]: Yeah, agreed.

We clearly need to support multiple devices but I think we would want to not start that work from where we are today!

**Comment by msackman-synthace** [comment]
Yeah, I think I'm in favour of pairing back to the bear minimum for now and then reconsidering the way forwards.

**Comment by haydnKing** [comment]
for archaeological purposes: antha-tester ran through fine

**Pull request merged**